### PR TITLE
Dedicated ServiceAccount for controller-manager

### PR DESCRIPTION
**Pull request by redbaron**

### CODE REVIEW

File: pkg/asset/internal/templates.go [hunk 1]
```diff
@@ -404,6 +404,7 @@ spec:
         command:
         - ./hyperkube
         - controller-manager
+        - --use-service-account-credentials
```
Comment (ericchiang) [inline pkg/asset/internal/templates.go]: fyi @colemickens this is a requirement for using pod security policies.

**Comment by ericchiang** [comment]
coreosbot test this please

**Comment by ericchiang** [comment]
Required for https://github.com/kubernetes-incubator/bootkube/issues/782

**Comment by redbaron** [comment]
Hello, I have more PRs in the pipeline, would like see first batch to be merged before submitting them to minimize rebasing

**Comment by colemickens** [comment]
coreosbot test this please

**Comment by colemickens** [comment]
@redbaron There are e2e failures here from Jenkins... It sounds like there were some improvements to e2e testing stability in the last seven days. Can you rebase this PR and ensure it passes the e2e tests and we can get it merged?

Thanks!

**Comment by redbaron** [comment]
coreosbot run e2e

**Comment by ericchiang** [comment]
coreosbot test this please

**Comment by colemickens** [comment]
coreosbot test this please

**Comment by redbaron** [comment]
coreosbot test this please

**Comment by ericchiang** [comment]
coreosbot test this please

**Comment by colemickens** [comment]
coreosbot test this please

**Comment by ericchiang** [comment]
coreosbot run e2e

**Comment by redbaron** [comment]
Soooo painful :(

**Comment by diegs** [comment]
coreosbot run e2e

**Comment by redbaron** [comment]
Can coreosbot results  be published somewhere visible to outsiders?

**Comment by diegs** [comment]
@redbaron will need to figure out a way to do that... it was a flake, I just filed:

**Comment by diegs** [comment]
coreosbot run e2e

**Comment by redbaron** [comment]
any chance to see it merged?

**Comment by diegs** [comment]
@redbaron sorry for the delay

File: pkg/asset/asset.go [hunk 2]
```diff
@@ -54,6 +54,8 @@ const (
 	AssetPathAPIServerSecret             = "manifests/kube-apiserver-secret.yaml"
 	AssetPathAPIServer                   = "manifests/kube-apiserver.yaml"
```
Comment (deltachonglee81) [inline pkg/asset/asset.go:55]: Agreed

File: pkg/asset/internal/templates.go (hunk 1 above)
Comment (deltachonglee81) [reply pkg/asset/internal/templates.go:406]: Sage []()

**Review by deltachonglee81 — COMMENTED** [review]
All good @

**Pull request merged**